src/main/java/dev/openfga/sdk/api/client/OpenFgaClient.java

```diff
@@ -273,19 +277,20 @@ public CompletableFuture<ClientWriteResponse> write(ClientWriteRequest request,
         configuration.assertValid();
         String storeId = configuration.getStoreIdChecked();
 
-        WriteRequest body = new WriteRequest();
+        if (options != null && !options.disableTransactions()) {
```

Comment:
In other [SDKs](https://github.com/openfga/js-sdk#write-create-and-delete-relationship-tuples) and also in our README, the default behaviour if options is not provided is transaction mode.

Reply:
```suggestion
        if (options == null || !options.disableTransactions()) {
```